hs_app_timeseries/receivers.py

```diff
@@ -35,9 +35,12 @@ def pre_add_files_to_resource_handler(sender, **kwargs):
     resource = kwargs['resource']
     files = kwargs['files']
     validate_files_dict = kwargs['validate_files']
-    fed_res_fnames = kwargs['fed_res_file_names']
+    source_names = kwargs['source_names']
 
-    if files or fed_res_fnames:
+    if __debug__:
+        assert(isinstance(source_names, list))
+
```

Comment:
@hyi @pkdash I would prefer to leave ones like this in temporarily, because in the next PR, I will use it as a sanity check that I am doing things correctly. This will prevent me from making common errors during porting. The __debug__ clauses are not invoked during production. 

What I am doing is difficult enough that I am triple-checking my code with __debug__ clauses that will catch errors I could make. I think anyone in my position would.